src/MovieCard.js

```diff
@@ -0,0 +1,13 @@
+import './MovieCard.css';
+
+const MovieCard = ({ img, title, rating }) => {
+  return (
+    <article className="movie-card">
+      <img className="movie-poster" src={img} alt="movie poster" />
```

Comment:
I'm glad to see yall including an alt tag - can you think of any information from the movie object we could use to make this alt tag more descriptive?